lib/raygun/raygun.rb

```diff
@@ -154,6 +154,7 @@ def update_ruby_version
     def initialize_git
       Dir.chdir(app_dir) do
         shell "git init"
+        shell "git checkout -q -b main"
```

Comment:
COOL!

Reply:
Do we need the -q also?

Reply:
The `-q` (quiet) is to keep the "switched to a new branch" message from cluttering up the output, i.e.

```
Checking for the latest application prototype... carbonfive/raygun-rails. Using cached version.
Switched to a new branch 'main'

Zap! Your application is ready. Next steps...
```

Reply:
Thanks for explaining!